UPGRADE-4.4.md

```diff
@@ -123,6 +123,50 @@ TwigBridge
 
  * Deprecated to pass `$rootDir` and `$fileLinkFormatter` as 5th and 6th argument respectively to the
    `DebugCommand::__construct()` method, swap the variables position.
+   
+TwigBundle
+----------
+
+ * Deprecated default value `twig.controller.exception::showAction` of the `twig.exception_controller` configuration option, set it to `null` instead
+ * Deprecated the `ExceptionController` and all built-in error templates, use the error renderer mechanism of the `ErrorRenderer` component
+ * Deprecated loading custom error templates in non-html formats: 
+
+  Before (`templates/bundles/TwigBundle/Exception/error.jsonld.twig`):
+  ```twig
+  { 
+      "@id": "https://example.com",
+      "@type": "error",
+      "@context": {
+          "title": "{{ status_text }}",
+          "code": {{ status_code }},
+          "message": "{{ exception.message }}"
+      }
+  }
+  ```
+
+  After (`App\ErrorRenderer\JsonLdErrorRenderer`):
+  ```php
+  class JsonLdErrorRenderer implements ErrorRendererInterface
```

Comment:
please mention that people need to register that as a service and use autoconfiguration or add a certain tag

Reply:
there is no autoconfiguration yet, we can add it later.